default.json

```diff
@@ -125,9 +125,9 @@
       "allowedVersions": "^8"
     },
     {
-      "description": "Prevent upgrade of dd-trace to v5.46.0",
+      "description": "Prevent upgrade of dd-trace to v5.46.0 and later versions",
       "matchPackageNames": ["dd-trace"],
-      "allowedVersions": "!/5.46.0/"
+      "allowedVersions": "<=5.45.0"
```

Comment:
Shouldn't this be `<=5.45`? Do we want to block fixes to `5.45`?

Reply:
Yes we want to block any new versions until we have more feedback from the usage of it, including 5.45 fixes
fyi here is another [PR](https://github.com/reside-eng/renovate-config/pull/99) where we'll pin it to 5.40.0 because there are reports from issues with 5.45 also: https://github.com/DataDog/dd-trace-js/issues/5554

Reply:
Also looking at their release activity they're never doing fixes on a past minor version so it shouldn't happen at all